packet/flatcar-linux/kubernetes/variables.tf

```diff
@@ -30,7 +30,7 @@ variable "os_channel" {
 variable "os_version" {
   type        = "string"
   default     = "current"
-  description = "Flatcar Linux version to install (for example '2191.5.0' - see https://www.flatcar-linux.org/releases/)"
+  description = "Flatcar Container Linux version to install (for example '2191.5.0' - see https://www.flatcar-linux.org/releases/), only for iPXE"
```

Comment:
Seems Flatcar Container Linux fix slipped into this commit.

Reply:
Also `operating_system` changes doesn't seem to be related to documentation update.